app/controllers/conferences_controller.rb

```diff
@@ -15,26 +15,33 @@ def show
   end
 
   def create
-    team = current_user.student_team
     @conference = Conference.new(conference_params)
     @conference.season_id = current_season.id
-    @conference.gid = generate_gid(team)
+    @conference.gid = generate_gid(current_user)
 
     if @conference.save
-      redirect_to edit_team_path(team), notice: 'Conference was successfully created.'
+      redirect
     else
       render action: :new
     end
   end
 
   def redirect
-    redirect_to orga_conferences_path
+    message = 'Conference was successfully created.'
+    case true
+    when current_user.current_student?
+      redirect_to edit_team_path(current_student.current_team), notice: message
+    when current_user.admin?
+      redirect_to orga_conferences_path, notice: message
+    else
+      redirect_to @conference, notice: message
+    end
   end
 
   private
 
-  def generate_gid(team)
-    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{team.id}"
+  def generate_gid(user)
+    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{user.id}"
```

Comment:
Hm, just wondering, does this have any impact on the conferences imported up to now?

Reply:
@branquinhoaa  says now we use 'user' instead 'team'.

Reply:
🤔  The change to :user instead of :team is implemented in another PR #828. So, it is better to not change it here. 👍